### PR TITLE
mempool & rpc log grooming

### PR DESCRIPTION
**Pull request by alexanderbez**
I have this set to be backported for 0.34.x. Obviously the small shim change will cause a conflict, but that'll be very easy to address.

### CODE REVIEW

**Review by marbar3778 — APPROVED** [review]

File: rpc/jsonrpc/server/http_json_handler.go [hunk 1]
```diff
@@ -97,8 +97,9 @@ func makeJSONRPCHandler(funcMap map[string]*RPCFunc, logger log.Logger) http.Han
 				}
 				args = append(args, fnArgs...)
 			}
+
 			returns := rpcFunc.f.Call(args)
-			logger.Info("HTTPJSONRPC", "method", request.Method, "args", args, "returns", returns)
```
[the remark this answers is not in the document]
Comment (alexanderbez) [reply rpc/jsonrpc/server/http_json_handler.go:101]: SAVAGE 😆

File: test/e2e/runner/rpc.go [hunk 2]
```diff
@@ -92,18 +92,22 @@ func waitForNode(node *e2e.Node, height int64, timeout time.Duration) (*rpctypes
...
-	lastHeight := int64(0)
+	var lastHeight int64
+
 	for _, node := range testnet.Nodes {
 		if node.Mode == e2e.ModeSeed {
 			continue
 		}
-		status, err := waitForNode(node, height, 20*time.Second)
+
+		status, err := waitForNode(node, height, 30*time.Second)
```
Comment (cmwaters) [inline test/e2e/runner/rpc.go:102]: I was going to implement something that depended on the size of the network i.e. more nodes = longer wait

**Review by cmwaters — APPROVED** [review]

File: test/e2e/runner/rpc.go (hunk 2 above)
Comment (alexanderbez) [reply test/e2e/runner/rpc.go:102]: Yeah, this should be configurable somewhat. Let's hold off on merging my PR until yours lands 👍

**Pull request merged**